.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["plugin:instawork/recommended"],
   "plugins": ["instawork"],
+  "ignorePatterns": ["**/*.ts", "**/*.tsx"],
```

Comment:
Is this temporary? 
We should not be ignoring since we have added eslint rules for TS also.